modules/apps/frontend-js/frontend-js-react-web/tsconfig.json

```diff
@@ -28,7 +28,8 @@
 	},
 	"include": [
 		"src/**/*",
-		"test/**/*"
+		"test/**/*",
+		"../../../global-env.d.ts"
```

Comment:
This will be now auto-generated with npm-scripts when this PR is released, https://github.com/liferay/liferay-frontend-projects/pull/790/files